docs/content/mixins/helm3.md

```diff
@@ -0,0 +1,172 @@
+---
+title: helm3 mixin
+description: Manage a Helm release with the helm v3 CLI
+aliases:
+- /mixins/helm/
+---
+
+<img src="/images/mixins/helm.svg" class="mixin-logo" style="width: 150px"/>
+
+This is a [Helm](https://helm.sh) v3 mixin for
+[Porter](https://github.com/getporter/porter). It executes the appropriate helm v3
+command based on which action it is included within: `install`, `upgrade`, or
+`delete`.
+
+Source: https://github.com/MChorfa/porter-helm3
+
+### Install or Upgrade
+
+```shell
+porter mixin install helm3 --feed-url https://mchorfa.github.io/porter-helm3/atom.xml
+```
+
+### Mixin Configuration
+
+Helm client version configuration. You can define others minors and patch versions up and down
+
+```yaml
+- helm3:
+    clientVersion: v3.3.4
+```
+
+Repositories
+
+```yaml
+- helm3:
+    repositories:
+      stable:
+        url: "https://charts.helm.sh/stable"
+```
+
+### Mixin Syntax
+
+Install
+
+```yaml
+install:
+  - helm3:
+      description: "Description of the command"
+      name: RELEASE_NAME
+      chart: STABLE_CHART_NAME
+      version: CHART_VERSION
+      namespace: NAMESPACE
+      replace: BOOL # Remove it if upsert is set to true. This is unsafe in production
+      devel: BOOL
+      wait: BOOL # default true
+      upsert: BOOL # default false. If set to true `upgrade --install` will be executed
+      set:
+        VAR1: VALUE1
+        VAR2: VALUE2
+```
+
+Upgrade
+
+```yaml
+install:
```

Comment:
```suggestion
upgrade:
```

Reply:
Thanks, I'll submit these changes upstream since it's text copied from the repo readme.